uvoz/uvoz.r

```diff
@@ -52,20 +52,26 @@ alkohol <- read_csv("podatki/who-alcohol.csv", skip=2,
 alkohol$vir <- NULL
 alkohol$tippijace <- NULL
 alkohol <- melt(alkohol, id.vars=c("drzava"), variable.name="leto", value.name="poraba")
-alkohol <- filter(alkohol,  !is.na(poraba))
+alkohol <- alkohol %>% drop_na()
+alkohol$leto <- alkohol$leto %>% parse_number()
+alkohol$razlog <- "alkohol"
 
 # Razširjenost kajenja tobačnih izdelkov 15+
 tobak <- read_csv("podatki/who-tabacco.csv", skip=2, 
-                  col_names=c("drzava", "leto", "moski", "zenske"), na="",
+                  col_names=c("drzava", "leto", "m", "z"), na="",
                   locale=locale(encoding = "UTF-8"))
-tobak <- filter(tobak,  moski!="")
-tobak$moski <- tobak$moski %>% strapplyc("^[0-9. ]+") %>% 
+tobak <- filter(tobak,  m!="")
+tobak$m <- tobak$m %>% strapplyc("^[0-9. ]+") %>% 
   unlist() %>% parse_number(locale=locale(decimal_mark=".", grouping_mark=" "))
-tobak$zenske <- tobak$zenske %>% strapplyc("^[0-9. ]+") %>% 
+tobak$z <- tobak$z %>% strapplyc("^[0-9. ]+") %>% 
   unlist() %>% parse_number(locale=locale(decimal_mark=".", grouping_mark=" "))
-tobak <- melt(tobak, id.vars=c("drzava", "leto"), measure.vars=c("moski", "zenske"), 
+tobak <- melt(tobak, id.vars=c("drzava", "leto"), measure.vars=c("m", "z"), 
               variable.name="spol", value.name="pojavnost")
+tobak$spol %>% parse_character()
+tobak$razlog <- "tobak"
 
+# Združena tabela za alkohol in tobak
+alktob <- inner_join(alkohol, tobak)
```

Comment:
Tole ti vrne prazno razpredelnico, saj `inner_join` privzeto združuje po vseh stolpcih z istim imenom - v tvojem primeru torej po državi, letu in razlogu. Ker se razlogi nikjer ne ujemajo, bo rezultat prazen. Če bi hotela združevati npr. po državi in letu, bi podala parameter `by = c("drzava", "leto")` (potem bi rezultat imel stolpca `razlog.x` in `razlog.y`).

Sicer pa teh dveh razpredelnic ni smiselno združevati, vsaj zaenkrat ne. Obe sta že v obliki *tidy data*, vendar imata različne stolpce - pri alkoholu nimaš spola, poleg tega pa imaš v eni tabeli stolpec `poraba`, v drugi pa `pojavnost`. Slednje bi morda res lahko razumela kot isti parameter - če bi pri obeh razpredelnicah imela spol (ali pri obeh ne), potem bi bilo morda smiselno združevati po vrsticah (`rbind`).

Združevanje z `inner_join` bi bilo smiselno, če bi hotela npr. narisati graf, ki prikazuje podatke iz obeh tabel v različnih dimenzijah (npr. `poraba` na osi x in `pojavnost` na osi y). Če boš to počela, lahko to združevanje opraviš takrat - še vedno pa bo treba rešiti problem, da v eni razpredelnici imaš podatke po spolu, v drugi pa ne.